pytorch_lightning/loops/fit_loop.py

```diff
@@ -51,15 +51,15 @@ def __init__(
         super().__init__()
         self.max_epochs = 1000 if (max_epochs is None and max_steps is None) else max_epochs
         self.min_epochs = 1 if (min_epochs is None and min_steps is None) else min_epochs
-        self.training_loop = TrainingEpochLoop(min_steps, max_steps)
-        self.validation_loop = EvaluationDataLoaderLoop()
+        self.epoch_loop = TrainingEpochLoop(min_steps, max_steps)
+        self.val_loop = EvaluationLoop()
```

Comment:
Still need to move this inside the epoch loop as we discussed.

Or do you want to do that in a separate PR?

Reply:
yes, I have it already implemented locally,
there is going to be a circular import due to that, so I would like to discuss the resolution of that in a separate PR.